<launch>
  <node name="joy_sound_player" pkg="ps4_joystick_sound_player" type="joy_sound_player_node" output="screen">
    <param name="hid_id" value="/dev/hidraw2" />
  </node>

  <node name="joy_node" pkg="joy" type="joy_node" args="_autorepeat_rate:=20" output="screen"/>    
</launch>